src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.quarto.component.spec.ts

```diff
@@ -1404,4 +1401,44 @@ describe('OnlineGameWrapperComponent of Quarto:', () => {
             tick(wrapper.joiner.maximalMoveDuration * 1000);
         }));
     });
+    describe('Visuals', () => {
+        it('should highlight each player name in their respective color', fakeAsync(async() => {
+            // given a game that has been started
+            await prepareStartedGameFor({ pseudo: 'creator', verified: true });
+            tick(1);
+            componentTestUtils.detectChanges();
+
+            // when the game is displayed
+
+            // then it should highlight the player's names
+            componentTestUtils.expectElementToHaveClass('#playerZeroIndicator', 'player0-bg');
+            componentTestUtils.expectElementToHaveClass('#playerOneIndicator', 'player1-bg');
+            tick(wrapper.joiner.maximalMoveDuration * 1000);
+        }));
+        it('should highlight the board with the color of the player when it is their turn', fakeAsync(async() => {
+            // given a game that has been started and for which it is the current player's turn
+            await prepareStartedGameFor({ pseudo: 'creator', verified: true });
+            tick(1);
+            componentTestUtils.detectChanges();
+
+            // when it is displayed
+
+            // then it should highlight the board with its color
+            componentTestUtils.expectElementToHaveClass('#board-tile', 'player0-bg');
+            tick(wrapper.joiner.maximalMoveDuration * 1000);
+        }));
+        it('should not highlight the board when it is the turn of the opponent', fakeAsync(async() => {
+            // given a game that has been started and for which it is not the current player's turn
+            await prepareStartedGameFor({ pseudo: 'creator', verified: true });
+            tick(1);
+            await doMove(FIRST_MOVE, true);
+            componentTestUtils.detectChanges();
+
+            // when it is displayed
```

Comment:
Haha, allez, tu pouvais plus facilement être poli avec le when là :D ! "when that's it, it's user's turn at last !" 
:D

Reply:
wat

Reply:
(je comprends que le when peut être amélioré mais je ne comprends pas ta proposition :D)

Reply:
// given a game that has been started and for which it is not the current player's turn
await prepareStartedGameFor({ pseudo: 'creator', verified: true });
tick(1);

// when it is his turn
await doMove(FIRST_MOVE, true);
componentTestUtils.detectChanges();

// then it should not highlight the board
componentTestUtils.expectElementNotToHaveClass('#board-tile', 'player1-bg');
tick(wrapper.joiner.maximalMoveDuration * 1000);


Pour une fois qu'on sais avoir un when :D !

Reply:
*when it is no longer his turn